docs/guides.md

```diff
@@ -6,7 +6,7 @@ title: How-to-Guides
 Practical step-by-step guides to help you achieve a specific goal. Most useful when you're trying to get something done. 
 
 ### Local Development
-- [How to spin up a DHIS2 local instance](./guides/spin-up-local-instance)
+- [How to spin up a DHIS2 local instance](./spin-up-local-instance)
```

Comment:
```suggestion
- [How to spin up a DHIS2 local instance](/docs/guides/spin-up-local-instance)
```

Reply:
Okay that's  great. Thanks for letting me know.